csharp/TheAccountClassInheritance/TheAccountClassInheritance/Accounts/PayingAccount.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TheAccountClassInheritance
+{
+    public class PayingAccount : Account
+    {
+        private const int _fee = 5;
```

Comment:
I'd suggest to make this constant public, so you can use it in the corresponding tests where you are asserting that the fees are applied correctly